uiowa/uiowa
# Classrooms - Migration Featured Image

Resolves #6395

# To Test

Sync D7 site in devdesktop (uiowa703)
Sync classrooms in D8: `ddev blt ds --site=classrooms.uiowa.edu`
`ddev composer install` if you haven't in awhile, or have been working on other branches
Enable the migrate module: `ddev drush @classrooms.local en classrooms_migrate`
Log in and visit the migrate config page: `ddev drush @classrooms.local uli /admin/config/sitenow/sitenow-migrate`
Complete the page using the form help text and devdesktop info
Check the migration status: `ddev drush @classrooms.local ms`
Import the rooms: `ddev drush @classrooms.local mim classrooms_room`
Go into the site, check out some of the newly migrated rooms, now with featured images.
